rules/sqlperf-reduce-table-size/rule.md

```diff
@@ -0,0 +1,36 @@
+---
+seoDescription: Do you keep your database tables small?
+type: rule
+archivedreason:
+title: Do you keep your database tables small?
+guid: cd4ffbd0-4a01-4229-9de3-c45207ebb07b
+uri: sqlperf-reduce-table-size
+created: 2024-07-23T07:26:54.0000000Z
+authors:
+  - title: Bryden Oliver
+    url: https://ssw.com.au/people/bryden-oliver
+related:
+  - sqlperf-select-required-columns
+  - sqlperf-verify-indexes-used
+  - sqlperf-avoid-implicit-type-conversions
+  - sqlperf-avoid-looping
+  - sqlperf-use-and-instead-of-or
+  - sqlperf-minimise-large-writes
+---
+
+Reading data from smaller tables is much faster. How can you keep the amount of data stored down.
+
+<!--endintro-->
+
+There are a number of reasons to keep table sizes small.
```

Comment:
@brydeno 

This sentence mentions "a number of reasons" but we only present a couple... can you reword?

Reply:
Done